pyhgvs/tests/genome.py

```diff
@@ -115,7 +119,7 @@ def read(self, filename):
 
         filename: a filename string or file stream.
         """
-        if isinstance(filename, basestring):
```

Comment:
I don't think that's _quite_ completely accurate. Both `str` (textual) and `bytes` (binary) can be used as filenames in both python 2 and python 3, but `six.string_types` does not include `bytes` in Python 3. IMO the most Pythonic way to write this would be to use duck typing:
```python
if hasattr(filename, 'read'):
    infile = filename
else:
    with open(filename) as infile:
        return self.read(infile)
```